### ISSUE TITLE
Missing test suite?

### ISSUE THREAD

**Issue by mara004** [internetarchive/archive-pdf-tools#50]
It looks like `archive-pdf-tools` currently does not have an automated test suite.
I know that not all developers like to work this way, but I think providing a test suite can be very advantageous for quality assurance, to make sure the library works equally well on different platforms. It may also be helpful to verify changes for correctness and avoid regressions.
`pytest` is a popular choice as a test framework among open-source Python projects, for instance.

**Comment by MerlijnWajer** [comment]
Agreed. I'm not much of a TDD guy, but it would be good to have tests.
I made some issues for that some time ago: https://github.com/internetarchive/archive-pdf-tools/issues/16 https://github.com/internetarchive/archive-pdf-tools/issues/5

but I haven't used pytest much at all, so I'd love some contributions from others if you/they feel up for it. :)
